recipes/ucsc-bedtobigbed/meta.yaml

```diff
@@ -1,21 +1,17 @@
 {% set package = "ucsc-bedtobigbed" %}
 {% set program = "bedToBigBed" %}
-{% set version = "447" %}
-{% set sha256 = "747a48486f7481d891e297baf63623b15d699265ede7339f654bcbc42481ac81" %}
+{% set version = "467" %}
 
 package:
   name: {{ package }}
   version: {{ version }}
 
 source:
-  url: "http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v{{ version }}.src.tgz"
-  sha256: {{ sha256 }}
-  patches:
-    - include.patch
-    - htmshell.patch
+  url: "https://github.com/ucscGenomeBrowser/kent/archive/refs/tags/v{{ version }}_base.tar.gz"
+  sha256: "27a139d71354645fd8366c82e7028a0b62212af0af4b2a407c1b000e28bbb4fd"
```

Comment:
What's the reason to not use the `sha256` variable ?
Many (all?!) `ucsc-XYZ` recipes use it and it is easier to keep them similar.

Reply:
I wasn't sure auto bump would work with that, and I was hoping to try the GitHub URL to avoid patches (which I read in some other comment) and maybe also allow the auto bumps to work. But the truth is I have no idea how the patches work, and since my simple ideas failed, I think I have to give up. Too bad. I have really been wanting an up-to-date version of this for a while since the UCSC tools can now read compressed beds.